File: DrinkUp.API/DrinkUp.DAL/Entities/Ponuda.cs
using System;
using System.Collections.Generic;

namespace DrinkUp.DAL.Entities
{
    public partial class Ponuda
    {
        public Ponuda()
        {
            Token = new HashSet<Token>();
        }

        public int Id { get; set; }
        public string Naslov { get; set; }
        public string Opis { get; set; }
        public double? Cijena { get; set; }
        public int BrojTokena { get; set; }
        public int VrstaPonudeId { get; set; }
        public int ObjektId { get; set; }
        public double? Popust { get; set; }

        public virtual Objekt Objekt { get; set; }
        public virtual VrstaPonude VrstaPonude { get; set; }
        public virtual ICollection<Token> Token { get; set; }
    }
}

File: MobilnaApp/app/src/main/java/com/example/drinkup/models/Ponuda.java
package com.example.drinkup.models;

import java.io.Serializable;

public class Ponuda implements Serializable {
    private int id;
    private String naslov;
    private String opis;
    private Float cijena;
    private int brojTokena;
    private int vrstaPonudeId;
    private int objektId;
    private Objekt objekt;
    private VrstaPonude vrstaPonude;

    public Ponuda() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNaslov() {
        return naslov;
    }

    public void setNaslov(String naslov) {
        this.naslov = naslov;
    }

    public String getOpis() {
        return opis;
    }

    public void setOpis(String opis) {
        this.opis = opis;
    }

    public Float getCijena() {
        return cijena;
    }

    public void setCijena(Float cijena) {
        this.cijena = cijena;
    }

    public int getBrojTokena() {
        return brojTokena;
    }

    public void setBrojTokena(int brojTokena) {
        this.brojTokena = brojTokena;
    }

    public int getVrstaPonudeId() {
        return vrstaPonudeId;
    }

    public void setVrstaPonudeId(int vrstaPonudeId) {
        this.vrstaPonudeId = vrstaPonudeId;
    }

    public int getObjektId() {
        return objektId;
    }

    public void setObjektId(int objektId) {
        this.objektId = objektId;
    }

    public Objekt getObjekt() {
        return objekt;
    }

    public void setObjekt(Objekt objekt) {
        this.objekt = objekt;
    }

    public VrstaPonude getVrstaPonude() {
        return vrstaPonude;
    }

    public void setVrstaPonude(VrstaPonude vrstaPonude) {
        this.vrstaPonude = vrstaPonude;
    }
}
